mailbox/postgres/src/test/java/org/apache/james/mailbox/jpa/JPASubscriptionManagerTest.java

```diff
@@ -30,14 +32,18 @@
 import org.apache.james.mailbox.SubscriptionManagerContract;
 import org.apache.james.mailbox.jpa.mail.JPAModSeqProvider;
 import org.apache.james.mailbox.jpa.mail.JPAUidProvider;
+import org.apache.james.mailbox.jpa.user.PostgresSubscriptionModule;
 import org.apache.james.mailbox.store.StoreSubscriptionManager;
 import org.apache.james.metrics.tests.RecordingMetricFactory;
-
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 class JPASubscriptionManagerTest implements SubscriptionManagerContract {
 
+    @RegisterExtension
+    static PostgresExtension postgresExtension = new PostgresExtension(PostgresSubscriptionModule.MODULE);
+
     static final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPAMailboxFixture.MAILBOX_PERSISTANCE_CLASSES);
```

Comment:
Hmmm I'm just realizing... That means we spawn a derby db for jpa test cluster, and a Postgresql for the new postgres mappers correct? 

Shouldn't we try to force jpa test cluster to point to a same single postgres server for testing? I'm not sure how hard or complex it is to achieve, but it would make more sense IMO. 

WDYT? If feasible could be an other ticket/PR of course.

Reply:
We spend cost for refactoring JpaTestCluster, and we will not use it in the future (when Postgres is complete).
But if we want to maintain jpa module, it is fine, I don't like derby, it looks old

Reply:
I share the same point with Tung. It is doable IMO, but until now I have not seen a clear benefit doing that yet. Or maybe I am wrong... :D

Reply:
Alright, fair point :)